README.md

```diff
@@ -5,118 +5,156 @@
 
-Once the codespace opens (this may take several minutes), open a new terminal.
+Once the codespace opens (this may take several minutes), open a new terminal and proceed to [deploy the app](#deploying-the-app).
 
 #### VS Code Dev Containers
+
 You can run the project in your local VS Code Dev Container using the [Dev Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers):
 
 1. Start Docker Desktop (install it if not already installed)
 2. Open the project:
 
     [![Open in Dev Containers](https://img.shields.io/static/v1?style=for-the-badge&label=Dev%20Containers&message=Open&color=blue&logo=visualstudiocode)](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/azure-samples/aisearch-openai-rag-audio)
-3. In the VS Code window that opens, once the project files show up (this may take several minutes), open a new terminal.
+3. In the VS Code window that opens, once the project files show up (this may take several minutes), open a new terminal, and proceed to [deploying the app](#deploying-the-app).
 
 #### Local environment
+
 1. Install the required tools:
-   - [Node.js](https://nodejs.org/en)
-   - [Python >=3.11](https://www.python.org/downloads/)
-      - **Important**: Python and the pip package manager must be in the path in Windows for the setup scripts to work.
-      - **Important**: Ensure you can run `python --version` from console. On Ubuntu, you might need to run `sudo apt install python-is-python3` to link `python` to `python3`.
-   - [Powershell](https://learn.microsoft.com/powershell/scripting/install/installing-powershell)
+   * [Azure Developer CLI](https://aka.ms/azure-dev/install)
+   * [Node.js](https://nodejs.org/)
+   * [Python >=3.11](https://www.python.org/downloads/)
+      * **Important**: Python and the pip package manager must be in the path in Windows for the setup scripts to work.
+      * **Important**: Ensure you can run `python --version` from console. On Ubuntu, you might need to run `sudo apt install python-is-python3` to link `python` to `python3`.
+   * [Git](https://git-scm.com/downloads)
+   * [Powershell](https://learn.microsoft.com/powershell/scripting/install/installing-powershell) - For Windows users only.
 
 2. Clone the repo (`git clone https://github.com/Azure-Samples/aisearch-openai-rag-audio`)
-4. The app needs to know which service endpoints to use for the Azure OpenAI and Azure AI Search. The following variables can be set as environment variables, or you can create a ".env" file in the "app/backend/" directory with this content.
-   ```
+3. Create a Python virtual environment and activate it.
```

Comment:
The 'start.sh/ps1' do this automatically now.

Reply:
Good point!